pages/chapter3.tsx

```diff
@@ -0,0 +1,71 @@
+import '../app/globals.css';
+import PageCard from '@/components/PageCard';
+import Layout from '@/components/Layout';
+import chapter from '@/constants/chapter.json';
+
+import Minimum from '@/components/exercises/chapter-3-oscar/Minimum';
+import Recursion from '@/components/exercises/chapter-3-oscar/Recursion';
+import BeanCounting from '@/components/exercises/chapter-3-oscar/BeanCounting';
+
+const chapterId = 3;
+
+const chapterMatch = chapter.find((chapter) => chapter.id === chapterId);
+
+if (!chapterMatch) {
+  throw new Error(`Chapter ${chapterId} not found`);
+}
+
+const { title, subtitle, details, devOscar, devJon } = chapterMatch;
+
+export default function Chapter3() {
```

Comment:
Can we add a test for Chapter3? it should go into `pageTests/Chapter3.test.tsx`

Reply:
Test added